server.py

```diff
@@ -22,7 +22,8 @@
 
 
 API_KEY = os.environ.get("GPTNEO", None)

```

Comment:
I think we're no longer using API_KEY, can you remove this line from this file and build.sh aswell >